geode-docs/tools_modules/gfsh/json_in_gfsh.html.md.erb

```diff
@@ -0,0 +1,80 @@
+---
+title:  Specifying JSON within Command-Line Options
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+Some `gfsh` commands allow JSON specifications within some options.
+These JSON strings specify pairs of strings.
+Each pair specifies one initialization property.
+
```

Comment:
I would rephrase this to say - "These JSON strings specify initialization properties in form of key/value pairs" or similar.

Reply:
+1